routes/index.js

```diff
@@ -1,55 +1,82 @@
-let express = require('express');
-let router = express.Router();
+let express = require('express')
+let router = express.Router()
+const dayjs = require('dayjs')
+const objectSupport = require("dayjs/plugin/objectSupport")
+dayjs.extend(objectSupport)
+const createNewHoliday = require('../db/create-new-holiday')
+const getAllHolidays = require('../db/get-all-holidays')
 
 /* GET home page. */
 router.get('/', async function(req, res, next) {
+  const holidayTable = []
+  const holidayData = await getAllHolidays()
+  const dateFormat = 'DD/MM/YYYY'
+
+  if (holidayData != undefined) {
+    holidayData.forEach(holiday => {
+      const startDate = dayjs(holiday.startDate)
+      const endDate = dayjs(holiday.endDate)
+
+      const projection = [
+          {
+            text: startDate.format(dateFormat)
+          },
+          {
+            text: endDate.format(dateFormat)
+          },
+          {
+            text: endDate.diff(startDate, 'day') + 1
+          }
+        ]
+      
+        holidayTable.push(projection)
+    })
+  }
+
   let data = {
```

Comment:
Is this still needed or was it just a placeholder? If it is still needed is it possible to make it into a const